google/cloud/bigtable/internal/defaults.cc

```diff
@@ -79,6 +79,13 @@ int DefaultConnectionPoolSize() {
 Options DefaultOptions(Options opts) {
   using ::google::cloud::internal::GetEnv;
 
+  if (opts.has<EndpointOption>()) {
+    auto const& ep = opts.get<EndpointOption>();
+    opts.set<DataEndpointOption>(ep);
```

Comment:
What if the application already provided `DataEndpointOption` for some reason?

Reply:
Seems to me this is a breaking change.  The behavior is now different.

Reply:
Fixed. PTAL